docs/docs/installation/installing-rasa-open-source.mdx

```diff
@@ -82,7 +82,25 @@ pip3 install rasa[full]
 
 to install all needed dependencies for every configuration.
 
-:::
```

Comment:
@miraai So sorry, just noticed this closing ::: of the note above is missing which means the page is weirdly formatted now, see: https://11817--rasahq-docs-rasa-v2.netlify.app/docs/rasa/installation/installing-rasa-open-source#additional-dependencies

Could you please add this closing punctuation above the Python 3.10 header? 🙏🏻